From dcbed7ddb673c5ce2e27f41f8e2c6b4dff9a3339 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 5 Dec 2019 09:39:52 +0800
Subject: [PATCH] ARM: rockchip_defconfig: disable CONFIG_KSM

Save memory and cpu usage for Android.
Base on android-recommended.config.

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: Id4ecfafab836df8a7a8ba382fa5f32e516176368
---
 arch/arm/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 8c89dc87ee8a..37673234f21c 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -90,7 +90,6 @@ CONFIG_MODVERSIONS=y
 CONFIG_PARTITION_ADVANCED=y
 CONFIG_CFQ_GROUP_IOSCHED=y
 # CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
-CONFIG_KSM=y
 CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
 CONFIG_CMA=y
 CONFIG_ZSMALLOC=y
-- 
2.35.3

